cypress/features/regression/designSystem/flatTable.feature

```diff
@@ -84,4 +84,19 @@ Feature: Design Systems FlatTable component
     Examples:
       | headerName | position |
       | Client     | first    |
-      | total      | second   |
\ No newline at end of file
+      | total      | second   |
+    
+  @posative
```

Comment:
Spelling mistake here. Should be `@Positive`.

Reply:
My speling has been bad lattely